waspc/examples/todo-typescript/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "prototype",
   "dependencies": {
     "wasp": "file:.wasp/out/sdk/wasp",
-    "@prisma/client": "4.16.2",
```

Comment:
> So it is not in their package.json because they are not using it directly?

Yes, exactly.

> What if they want to run npx prisma -> they can't right?

I think they can. They do have the Prisma CLI, they just don't have the client. Why do you think it wouldn't work?

> I mean even if they can it won't use our version of Prisma. Is that bad or good?

I'm not following this one. I think there's only a single version of Prisma. Which versions are you considering?

Reply:
Sorry, I mixed prisma cli and prisma client, all good.